### ISSUE TITLE
Serious flake8 issues

### ISSUE THREAD

**Issue by adivijaykumar** [gwnrtools/nr-catalog-tools#19]
Here's a summary of the flake8 errors introduced by #18. This needs to be fixed on priority.

```
./nrcatalogtools/lvc.py:454:5: F841 local variable 'LNhat' is assigned to but never used
./nrcatalogtools/lvc.py:458:5: F841 local variable 'nhat' is assigned to but never used
./nrcatalogtools/lvc.py:462:19: F821 undefined name 'LNhatx'
./nrcatalogtools/lvc.py:463:19: F821 undefined name 'LNhaty'
./nrcatalogtools/lvc.py:464:19: F821 undefined name 'LNhatz'
./nrcatalogtools/lvc.py:465:18: F821 undefined name 'nhatx'
./nrcatalogtools/lvc.py:466:18: F821 undefined name 'nhaty'
./nrcatalogtools/lvc.py:467:18: F821 undefined name 'nhatz'
./nrcatalogtools/lvc.py:496:5: F841 local variable 'IntParams' is assigned to but never used
./nrcatalogtools/lvc.py:496:17: F821 undefined name 'GetInterpRefValuesFromH5File'
./nrcatalogtools/lvc.py:499:13: F821 undefined name 'RefParams'
./nrcatalogtools/lvc.py:499:47: F821 undefined name 'RefParams'
./nrcatalogtools/lvc.py:500:13: F821 undefined name 'RefParams'
./nrcatalogtools/lvc.py:500:47: F821 undefined name 'RefParams'
./nrcatalogtools/lvc.py:501:13: F821 undefined name 'RefParams'
./nrcatalogtools/lvc.py:501:47: F821 undefined name 'RefParams'
./nrcatalogtools/lvc.py:505:5: F841 local variable 'nhat' is assigned to but never used
./nrcatalogtools/lvc.py:509:15: F821 undefined name 'RefParams'
./nrcatalogtools/lvc.py:510:15: F821 undefined name 'RefParams'
./nrcatalogtools/lvc.py:511:15: F821 undefined name 'RefParams'
./nrcatalogtools/lvc.py:519:19: F821 undefined name 'LNhatx'
./nrcatalogtools/lvc.py:520:19: F821 undefined name 'LNhaty'
./nrcatalogtools/lvc.py:521:19: F821 undefined name 'LNhatz'
./nrcatalogtools/lvc.py:522:18: F821 undefined name 'nhatx'
./nrcatalogtools/lvc.py:523:18: F821 undefined name 'nhaty'
./nrcatalogtools/lvc.py:524:18: F821 undefined name 'nhatz'
./nrcatalogtools/lvc.py:670:5: F841 local variable 'tol' is assigned to but never used
./nrcatalogtools/lvc.py:698:5: F841 local variable 'ReqDefAttrsSXS' is assigned to but never used
./nrcatalogtools/lvc.py:720:24: F821 undefined name 'GetRefTimeFromRefFreq'
./nrcatalogtools/lvc.py:722:42: F821 undefined name 'CheckInterpReq'
./nrcatalogtools/lvc.py:730:34: F821 undefined name 'CheckInterpReq'
./nrcatalogtools/lvc.py:737:16: F821 undefined name 'GetRefTimeFromMetadata'
./nrcatalogtools/lvc.py:744:40: F821 undefined name 'CheckNRAttrs'
./nrcatalogtools/lvc.py:754:48: F821 undefined name 'CheckNRAttrs'
./nrcatalogtools/lvc.py:760:33: F821 undefined name 'ComputeLALSourceFrameFromSXSMetadata'
./nrcatalogtools/lvc.py:768:29: F821 undefined name 'GetRefVals'
./nrcatalogtools/lvc.py:771:25: F821 undefined name 'GetRefVals'
./nrcatalogtools/lvc.py:792:48: F821 undefined name 'CheckNRAttrs'
./nrcatalogtools/lvc.py:800:25: F821 undefined name 'ComputeLALSourceFrameByInterp'
./nrcatalogtools/lvc.py:913:5: F841 local variable 'salpha' is assigned to but never used
```

**Comment by prayush** [comment]
@vaishakp I get the same errors after merging all but #3 #13 

```
$ flake8 nrcatalogtools
nrcatalogtools/lvc.py:454:5: F841 local variable 'LNhat' is assigned to but never used
nrcatalogtools/lvc.py:458:5: F841 local variable 'nhat' is assigned to but never used
nrcatalogtools/lvc.py:462:19: F821 undefined name 'LNhatx'
nrcatalogtools/lvc.py:463:19: F821 undefined name 'LNhaty'
nrcatalogtools/lvc.py:464:19: F821 undefined name 'LNhatz'
nrcatalogtools/lvc.py:465:18: F821 undefined name 'nhatx'
nrcatalogtools/lvc.py:466:18: F821 undefined name 'nhaty'
nrcatalogtools/lvc.py:467:18: F821 undefined name 'nhatz'
nrcatalogtools/lvc.py:496:5: F841 local variable 'IntParams' is assigned to but never used
nrcatalogtools/lvc.py:496:17: F821 undefined name 'GetInterpRefValuesFromH5File'
nrcatalogtools/lvc.py:499:13: F821 undefined name 'RefParams'
nrcatalogtools/lvc.py:499:47: F821 undefined name 'RefParams'
nrcatalogtools/lvc.py:500:13: F821 undefined name 'RefParams'
nrcatalogtools/lvc.py:500:47: F821 undefined name 'RefParams'
nrcatalogtools/lvc.py:501:13: F821 undefined name 'RefParams'
nrcatalogtools/lvc.py:501:47: F821 undefined name 'RefParams'
nrcatalogtools/lvc.py:505:5: F841 local variable 'nhat' is assigned to but never used
nrcatalogtools/lvc.py:509:15: F821 undefined name 'RefParams'
nrcatalogtools/lvc.py:510:15: F821 undefined name 'RefParams'
nrcatalogtools/lvc.py:511:15: F821 undefined name 'RefParams'
nrcatalogtools/lvc.py:519:19: F821 undefined name 'LNhatx'
nrcatalogtools/lvc.py:520:19: F821 undefined name 'LNhaty'
nrcatalogtools/lvc.py:521:19: F821 undefined name 'LNhatz'
nrcatalogtools/lvc.py:522:18: F821 undefined name 'nhatx'
nrcatalogtools/lvc.py:523:18: F821 undefined name 'nhaty'
nrcatalogtools/lvc.py:524:18: F821 undefined name 'nhatz'
nrcatalogtools/lvc.py:670:5: F841 local variable 'tol' is assigned to but never used
nrcatalogtools/lvc.py:698:5: F841 local variable 'ReqDefAttrsSXS' is assigned to but never used
nrcatalogtools/lvc.py:720:24: F821 undefined name 'GetRefTimeFromRefFreq'
nrcatalogtools/lvc.py:722:42: F821 undefined name 'CheckInterpReq'
nrcatalogtools/lvc.py:730:34: F821 undefined name 'CheckInterpReq'
nrcatalogtools/lvc.py:737:16: F821 undefined name 'GetRefTimeFromMetadata'
nrcatalogtools/lvc.py:744:40: F821 undefined name 'CheckNRAttrs'
nrcatalogtools/lvc.py:754:48: F821 undefined name 'CheckNRAttrs'
nrcatalogtools/lvc.py:760:33: F821 undefined name 'ComputeLALSourceFrameFromSXSMetadata'
nrcatalogtools/lvc.py:768:29: F821 undefined name 'GetRefVals'
nrcatalogtools/lvc.py:771:25: F821 undefined name 'GetRefVals'
nrcatalogtools/lvc.py:792:48: F821 undefined name 'CheckNRAttrs'
nrcatalogtools/lvc.py:800:25: F821 undefined name 'ComputeLALSourceFrameByInterp'
nrcatalogtools/lvc.py:913:5: F841 local variable 'salpha' is assigned to but never used
```
Can you have a look pls?

**Comment by vaishakp** [comment]
I hadn't yet pushed all the changes when the pull request #18 was merged. Please check now.

**Comment by prayush** [comment]
Resolved by #3
